app/grandchallenge/archives/views.py

```diff
@@ -590,14 +598,11 @@ def new_interface_url(self):
             kwargs={"slug": self.base_object.slug},
         )
 
-    def process_data_for_object(self, data):
-        """Creates an archive item"""
-        instance = ArchiveItem.objects.create(archive=self.base_object)
-        for slug in data:
-            instance.create_civ(
-                ci_slug=slug, new_value=data[slug], user=self.request.user
-            )
-        return instance
+    def create_object(self, non_civ_data):
+        return ArchiveItem.objects.create(
+            archive=self.base_object,
+            **non_civ_data,
+        )
```

Comment:
Don't you have a method for that on the base object now that you could reuse here?

Reply:
You are right, but in this case this was a left over, good find!